akka-multi-node-testkit/src/main/scala/akka/remote/testkit/MultiNodeSpec.scala

```diff
@@ -166,7 +166,7 @@ object MultiNodeSpec {
    * -Dmultinode.tcp.port=0
    * }}}
    */
-  val tcpPort: Int = Integer.getInteger("multinode.tpc.port", 0)
+  val tcpPort: Int = Integer.getInteger("multinode.tcp.port", 0)
```

Comment:
That silly typo wasted hours of my life and damage my mental sanity. I'm ok now.